jquery.flot.absRelTime.js

```diff
@@ -182,9 +360,16 @@ the the second one the date in gregorian date format.
             var c = fmt.charAt(i),
                 localDateValue = d.date || d.getDate();
             if (escape) {
+                var timeFormatStartIndex = fmt.indexOf('<');
```

Comment:
Oh, so the angle brackets _are_ a required part of the format string!  Please disregard my earlier doc comment.